cmd/watch.go

```diff
@@ -80,6 +83,9 @@ func watch(ctx *cmdutil.Ctx, events chan file.Event, sig chan os.Signal) error {
 			}
 			ctx.Log.Printf("[%s] processing %s", colors.Green(ctx.Env.Name), colors.Blue(event.Path))
 			perform(ctx, event.Path, event.Op, event.LastKnownChecksum)
+			if event.Op != file.Skip && event.Op != file.Remove {
```

Comment:
Might we still want `remove` to trigger a notification? e.g. if you delete an image that seems to be used, you may want to still reload the page to ensure nothing has broken.